com.snipaste.Snipaste.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.snipaste.Snipaste</id>
+  <name>Snipaste</name>
+  <project_license>LicenseRef-proprietary=https://www.snipaste.com/index.html</project_license>
+  <developer_name>Snipaste</developer_name>
+  <summary>Snipaste is not just a snipping tool. It will boost your productivity if you get used to its 'paste'!</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <launchable type="desktop-id">com.snipaste.Snipaste.desktop</launchable>
```

Comment:
is redistribution allowed?

Reply:
waiting for official reply

Reply:
Will this block the redistribution?

Reply:
I‘m the author of Snipaste. I approve this redistribution. Thank you @chenzhiwei !

Reply:
thanks. do you want to be added to the repo as a collaborator if/when this is merged?